tensorboard/plugins/graphs/tf_graph_loader/tf-graph-loader.html

```diff
@@ -61,6 +61,7 @@
       readOnly: true,
       notify: true
     },
+    /** @type {Object?} */
```

Comment:
Indeed, the compiler is happy after removing the `?`. This seems somewhat confusing to me: clearly the compiler is inferring types from the Polymer property definitions to some degree, and yet a property with `type: Object` still needs to be annotated as `/** @type {Object} */`. Ah, well.